# SPDX-License-Identifier: MPL-2.0
# libdatachannel 0.23.2: FreeBSD 5.1 math.h predates C99 round(); use the
# std::round overload (the file already includes <cmath>).
--- a/src/rtppacketizationconfig.cpp
+++ b/src/rtppacketizationconfig.cpp
@@ -44,7 +44,7 @@
 }
 
 uint32_t RtpPacketizationConfig::getTimestampFromSeconds(double seconds, uint32_t clockRate) {
-	return uint32_t(int64_t(round(seconds * double(clockRate)))); // convert to integer then cast to u32
+	return uint32_t(int64_t(std::round(seconds * double(clockRate)))); // convert to integer then cast to u32
 }
 
 uint32_t RtpPacketizationConfig::secondsToTimestamp(double seconds) {
